### ISSUE TITLE
EcAccess Jobs queuing forever

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1030]
In GitLab by @dbeltrankyl on May 2, 2023, 12:15

This is an issue in the ecaccess part, 

Sometimes the job status is in QUEUING forever, even if they ran. 

```
$ ecaccess-job-list 21298094
     Job-Id: 21298094
   Job Name: a5tg_19900101_fc00_INI.cmd
      Queue: hpc
       Host: hpc-batch.ecmwf.int
   Schedule: May 02 09:27
 Expiration: May 09 09:27
  Try Count: 1/1
     Status: WAIT
```

From Autosubmit, we could mitigate it by checking if the job has generated logs in the remote. 

This checking should be activated manually from the configuration parameters and by the owner of the experiment. 

What Autosubmit should do is: 

* Check if the new parameter: Check_job_in_queue_timeout > 00:00  ( name TODO )

* Each $TIME, check if .0 logs exists and COMPLETE file exists. 

* If both files exist, mark the job as completed, else, check the creation date of the log and compare it with current time and wallclock specified. 

FYI: @gmontanepinto

**Comment by kinow** [comment]
In GitLab by @gmontanepinto on May 4, 2023, 18:08

I did some tests and I found that this is only happening when the interactive queue is used (`ni` queue on HPC2020). We actually don't need to use this interactive queue but the `nf` and `np` queues which are working fine, so there's no real need to implement these new checks.

I've contacted ECMWF support and they will check if they can improve the jobs monitoring with ECaccess.
